.cirrus.yml

```diff
@@ -0,0 +1,9 @@
+freebsd_instance:
```

Comment:
can you add a file header to this file like we have in the travis one ?

Reply:
will do.